dev/preview/BUILD.yaml

```diff
@@ -15,7 +15,7 @@ scripts:
   - name: get-credentials
     description: Provisions a new preview environment
     script: |
-      previewctl get-credentials --kube-save-path $HOME/.kube/config
+      previewctl get-credentials KUBECONFIG=$HOME/.kube/config
```

Comment:
The `KUBECONFIG=$HOME/.kube/config` should be before the command execution, but it looks like this leeway script isn't used anywhere and the description of it is wrong, so we can clean this up some other time ☺️

Reply:
fixed